interface/wx/bmpbndl.h

```diff
@@ -246,15 +246,18 @@ class wxBitmapBundle
     /**
         Create a bundle from the SVG image.
 
-        Please note that the current implementation uses NanoSVG library
-        (https://github.com/memononen/nanosvg) for parsing and rasterizing SVG
-        images which imposes the following limitations:
+        Please note that the current implementation uses either LunaSVG library
+        (https://github.com/sammycage/lunasvg) (since wxWidgets 3.3) or NanoSVG
+        library (https://github.com/memononen/nanosvg) (since wxWidgets 3.1.7)
+        for parsing and rasterizing SVG images which imposes the following
+        limitations:
 
         - Text elements are not supported at all (see note for workaround).
         - SVG 1.1 filters are not supported.
         - Embedded images are not supported (see note for workaround).
 
-        These limitations will be relaxed in the future wxWidgets versions.
+        These limitations may be relaxed in future wxWidgets versions (they are
```

Comment:
Minor, but I'd say
```suggestion
        These limitations may be relaxed in future wxWidgets versions (these features are
```
to avoid implying that limitations are planned in LunaSVG :-)